docs/Overview.md

```diff
@@ -10,7 +10,7 @@ There are several different concepts that make up the complete query engine.
     A query is represented as a directed acyclic graph (DAG).
 * Flux - Functional Language for defining a query to execute.
 * Parser - Parses a Flux script and produces a query.
-* Data Frame - A data frame is a matrix of time series data where one dimension is time and the other is series.
+* Table - Data is modeled as a sequence of records in table.
```

Comment:
"in table" seems like there is a missing word but I can't identify which one was intended. Also, is the intention that table now represents all data in a single schema (like the tableflux proposal) or are we intending to capture the current state where a table is a sequence of records with a common schema and group key?

Reply:
Seems to be a circular definition? Thinking this should be a simple definition such as a "table is a list of rows" ('m sure there is something better floating around, just an example)